src/utils.ts

```diff
@@ -35,7 +35,7 @@ export function formatNanoseconds(value: number | bigint): string {
 }
 
 export function getNanosecondsTime(): bigint {
-  if (typeof process !== "undefined") {
+  if (typeof process !== "undefined" && process.hrtime !== undefined) {
```

Comment:
it feels a little inconsistent here... do we need an issue to use `!== undefined` everywhere? Using `typeof x !== "string"` feels kind of old school, right???

Reply:
The first part of the conditional makes sense because, if `process` is not declared (as opposed to be `undefined`, then comparing it against undefined, or trying to do something like `if (process)` raises an error.

It is true, though, that the second part of the conditional could have been simplified.